fast_llm/layers/language_model/config.py

```diff
@@ -57,6 +61,12 @@ class LanguageModelArchitectureConfig(BaseModelArchitectureConfig):
     tie_word_embeddings: bool = Field(
         default=True, desc="Tie the output weights (logits) with the vocabulary embedding.", hint=FieldHint.core
     )
+    num_multi_token_prediction_heads: int | None = Field(
+        default=None,
+        desc="Number of multi-token prediction heads.",
+        hint=FieldHint.feature,
+        valid=skip_valid_if_none(check_field(Assert.gt, 0)),
```

Comment:
1 is a weird case where we just add an extra transformer layer to a standard LM for some reason, how about `skip_valid_if_none(check_field(Assert.gt, 1))`?

Reply:
`num_multi_token_prediction_heads=1` is the default now, to behave like a standard LM. So keeping this condition, and removed the `skip_valid_if_none`